public/index.html.erb

```diff
@@ -142,6 +142,7 @@
             </div>
             <div class="col-lg-4 col-md-4">
               <p class="fork small-text text-center"><a href="https://github.com/dblock/slack-amber-alert" target="_blank">fork me on github</a> | <a href='/api'>RESTful API</a></p>
+ <p class="fork small-text text-center"><a href="privacy.html" target="_blank">Privacy Policy</a></p>
```

Comment:
Right. In Sublime text I use HTMLBeautify or some other thing like that and it cleans up my HTML for me. Do this for all pages so that the diffs are clean and we know what's going on.